common-utils/src/main/java/com/stnetix/ariaddna/commonutils/DTO/vufs/ChunkFile.java

```diff
@@ -0,0 +1,27 @@
+package com.stnetix.ariaddna.commonutils.DTO.vufs;
+
+import java.io.Serializable;
+
+/**
+ * Describe the chunk of file
+ */
```

Comment:
It is quite obvious that the name of the class speaks for itself, but I suggest you to write a detailed description about what a developer should use this class for.